iOS/rainbow/Model/KituraKit/Client.swift

```diff
@@ -73,10 +90,14 @@ public class KituraKit {
     /// - Parameter route: The custom route KituraKit points to during REST requests.
     public func get<O: Codable>(_ route: String, respondWith: @escaping CodableResultClosure<O>) {
         let url = baseURL.appendingPathComponent(route)
-        RestRequest(url: url.absoluteString)
-          .handle(respondWith)
+        let request = RestRequest(url: url.absoluteString)
+        guard let config = loadedCredentials else {
+            return
+        }
+        request.credentials = Credentials.basicAuthentication(username: config.username, password: config.password)
```

Comment:
Note for the Kitura team: there was no way to add credentials (or any header) to the request from the interface of KituraKit. We should consider fixing this!!!